static/external/manyplot.js

```diff
@@ -18,10 +18,67 @@ function getNearest(arr,value) {
     return index;
```

Comment:
I don't have too much to say on manyplot.js.  You're probably far more familiar with it at this point than I am.  Looks good though.  I'll try to schedule in some testing to try to break all of this if I get shifted back to Dataflow.